addons/project/models/project.py

```diff
@@ -911,11 +919,82 @@ def _show_profitability(self):
     def _get_profitability_aal_domain(self):
         return [('account_id', 'in', self.analytic_account_id.ids)]
 
-    def _get_profitability_items(self, with_action=True):
+    def action_aal(self, domain=None, res_id=False):
+        self.ensure_one()
+        action = self.env["ir.actions.actions"]._for_xml_id("analytic.account_analytic_line_action_entries")
+        action['domain'] = domain or []
+        action['context'] = {
+            'grid_anchor': (datetime.now()).strftime('%Y-%m-%d'),
+            'grid_range': 'year',
```

Comment:
Indeed, i can just get rid of the 2 grid args, since it seems it is now the default for the grid view, but it seems there's another problem : the grid view does not care about the domain given. It does not show the expected results (all months have a value of 0), and when the user click on the magnifying glass to see the detail, it shows all the aal, for the month selected, and not only the ones from the domain. I'm still looking to find the reason and a solution.

Reply:
edit, it seems to be more generic bug. Investigation and solution can be followed on this task : https://www.odoo.com/web#id=3254529&menu_id=4720&cids=1&action=333&active_id=4105&model=project.task&view_type=form